problemtools/verifyproblem.py

```diff
@@ -1687,6 +1686,8 @@ def validate(self, testcase: TestCase, submission_output: str) -> SubmissionResu
                 shutil.rmtree(validator_output)
                 if res.verdict != 'AC':
                     return res
+            else:
+                self.warning(f"Compilation failed for {val.name}: {compile_res[1]}")
```

Comment:
Won't this result in a lot of warning spam (one log line per submission per test case)? IIRC we already give an error in an earlier phase.

Reply:
I had an error and got no information before adding this. It was very confusing.
I will double check though.